third_party/move/move-compiler/src/parser/syntax.rs

```diff
@@ -2522,6 +2574,43 @@ fn parse_ability(context: &mut Context) -> Result<Ability, Box<Diagnostic>> {
     }
 }
 
+// Parse an optional "with" type constraint:
+//      WithAbilities =
+//          ( "with" <Ability> (+ <Ability>)* )?
+fn parse_with_abilities(context: &mut Context) -> Result<Vec<Ability>, Box<Diagnostic>> {
```

Comment:
In the PR description (at the very beginning), it says syntax is extended with the following: `move |args| body with copy, store`. But it looks like we use `+` (not `,`) as the separator for abilities in both lambda expressions and types. Just double checking that this is what is intended and the PR description was a typo.